### PR TITLE
Add text resize, respacing and zoom doc page

### PR DESCRIPTION
**Pull request by khiga8**
Resolves https://github.com/github/accessibility/issues/804

This PR introduce a common pattern accessibility page related to text resizing, text respacing, and testing with zoom.

### CODE REVIEW

File: content/accessibility/text-resize-and-respacing.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Modify text spacing according to the [WCAG text spacing testing guidelines](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html). You can do this
+by changing the CSS using the browser inspector or by using a tool like [Steve Faulkner: text spacing bookmarklet](https://cdpn.io/stevef/debug/YLMqbo).
+
+Observe that text is contained within the bounds of it's container without overlapping or being cutoff.
+
+## Guidelines
+
+### For designers
+
```
Comment (khiga8) [inline content/accessibility/text-resize-and-respacing.mdx:39]: Hey @langermank and @vdepizzol! I heard that you're doing work related to Primer font-size conversion to REM system. Is there any guidance you think we should offer here to designers and developers meanwhile?

File: content/accessibility/text-resize-and-respacing.mdx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    <img width="600" alt="Screenshot of a GitHub issue description with adjusted text spacing. The issue description is contained within the bounds of the container even with adjusted text spacing " src="https://user-images.githubusercontent.com/16447748/157532517-38ae80a0-f5f8-47ec-b9fc-7d5ba5d27bd2.png"/>
+  </Do>
+  <Do>
+    <Caption>Do use the browser zoom functionality to verify that the page is readable and functional, and make adjustments if necessary</Caption>
+    <img width="800" alt="Screenshot of a GitHub markdown toolbar that becomes unusable with overlapping controls at browser magnification of 200%" src="https://user-images.githubusercontent.com/16447748/157534278-daa67dbd-ecfb-4964-ab45-13d6037355a0.png"/>
+  </Do>
+  <Dont>
+    <Caption>Don't use a fixed height on text content containers</Caption>
+    <img width="600" alt="Screenshot of a GitHub issue description where the container has a fixed height, causing the authored long description to overflow out of bounds" src="https://user-images.githubusercontent.com/16447748/157533237-a18c1cfb-d962-4349-8cbd-a390174b4ebc.png"/>
```
Comment (inkblotty) [inline content/accessibility/text-resize-and-respacing.mdx:63]: In these images, the detailed description in the Alt text could be useful for sighted people too. Can we add to the Caption? For example:
`Don't use a fixed height on text content containers. In the image below, you can see that the fixed height of the container causes it to overflow out of bounds.`

**Review by inkblotty — COMMENTED** [review]
Can we make some of the useful `alt` text a caption for sighted users who may not know what's significant about the image?

File: content/accessibility/text-resize-and-respacing.mdx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+Users should be able to resize text and use custom text spacing without text being clipped, truncated, or obscured. In addition, interactive controls should not
+break.
+
+## Why?
+
+People with low-vision may rely on the browser's zoom functionality or update the font-size of a page to read content. Additionally, people with low-vision
+and people with dyslexia may increase text spacing with custom CSS to improve readability. 
+
+When sites are built without adequate responsivity considerations (e.g. setting fixed width and height, using non-relative font sizing), sites may become completely unreadable
```
Comment (yaili) [inline content/accessibility/text-resize-and-respacing.mdx:15]:
```suggestion
When sites are built without adequate responsivity considerations (for example: setting fixed width and height, using non-relative font sizing), sites may become completely unreadable 
```

Is _responsivity_ a word?

File: content/accessibility/text-resize-and-respacing.mdx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+and people with dyslexia may increase text spacing with custom CSS to improve readability. 
+
+When sites are built without adequate responsivity considerations (e.g. setting fixed width and height, using non-relative font sizing), sites may become completely unreadable 
+and unusable for some users.
+
+## How to test 
+
+### Text resizing
+
+Most modern browsers support full-page zoom, while some (e.g. FireFox) also support text-only zoom. While WCAG does not specify how text should be resized, it is best practice for your
```
Comment (yaili) [inline content/accessibility/text-resize-and-respacing.mdx:22]:
```suggestion
Most modern browsers support full-page zoom, while some also support text-only zoom (for example: Firefox). While WCAG does not specify how text should be resized, it is best practice for your 
```

File: content/accessibility/text-resize-and-respacing.mdx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+and unusable for some users.
+
+## How to test 
+
+### Text resizing
+
+Most modern browsers support full-page zoom, while some (e.g. FireFox) also support text-only zoom. While WCAG does not specify how text should be resized, it is best practice for your 
+page to be readable with full-page zoom and text-only zoom. Aside from using the browser zoom, some users may also resize text through browser font-size setting.
+
+Test your page by using browser zoom with magniciation of up to 200%. Observe that the text content does not overflow, overlap, or become truncated, and that interactive controls still work.
```
Comment (yaili) [inline content/accessibility/text-resize-and-respacing.mdx:25]:
```suggestion
Test your page by using browser zoom with magnification of up to 200%. Confirm that the text content does not overflow, overlap, or become truncated, and that interactive controls still work.
```

File: content/accessibility/text-resize-and-respacing.mdx [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+Learn more about resize testing techniques at [TheA11yProject: Resize text](https://www.a11yproject.com/posts/resize-text/). 
+
+#### Text spacing
```
Comment (yaili) [inline content/accessibility/text-resize-and-respacing.mdx:29]:
```suggestion
### Text spacing
```

File: content/accessibility/text-resize-and-respacing.mdx [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+Modify text spacing according to the [WCAG text spacing testing guidelines](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html). You can do this
+by changing the CSS using the browser inspector or by using a tool like [Steve Faulkner: text spacing bookmarklet](https://cdpn.io/stevef/debug/YLMqbo).
+
+Observe that text is contained within the bounds of it's container without overlapping or being cutoff.
```
Comment (yaili) [inline content/accessibility/text-resize-and-respacing.mdx:34]:
```suggestion
Confirm that text is contained within the bounds of its container without overlapping or being cutoff.
```

**Review by yaili — CHANGES_REQUESTED** [review]

**Comment by khiga8** [comment]
@inkblotty, @yaili thank you for the thorough review 🙇‍♀️

**Review by yaili — APPROVED** [review]
👍🏼

**Pull request merged**